app/views/layouts/application.html.erb

```diff
@@ -23,11 +21,12 @@
       <%= yield %>
     </main>
     <%= render partial: "components/footer" %>
-    <%= render partial: "components/tracking_scripts" %>
+    <script src="/tradingview/charting_library.min.js"></script>
```

Comment:
great idea to move scripts to the footer! another consideration is to make these asynchronous, rather than using synchronous script references. it would also be interesting to see the performance profile report from before and after. 

example: 
```js
  (function () {
        var s = document.createElement('script');
        s.type = 'text/javascript';
        s.async = true;
        s.src = /tradingview/charting_library.min.js';
        var x = document.getElementsByTagName('script')[0];
        x.parentNode.insertBefore(s, x);
    })();
```